client/src/components/commonComponents/Input/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
+
+const Input = ({ type, name, label, placeholder, value, onChange }) => {
+  return (
+    <input
+      type={type}
+      name={name}
+      aria-label={label}
+      placeholder={placeholder}
+      value={value}
+      onChange={onChange}
```

Comment:
how can I make the style to th input --> add className attr

Reply:
and add span for the error msg

Reply:
I added both